pages/api-reference/funding-sources/remove.mdx

```diff
@@ -13,9 +13,9 @@ Remove a funding source by id. A removed funding source is soft deleted and can
 
 ##### Best Practices
 
-If a funding-source has an **outgoing** transfer, we recommend not removing it until the transfer has been [exported](https://developers.dwolla.com/concepts/transfer-processing-times#transfer-processing-times). Doing so will result in the cancellation of the transfer. Determining if a transfer has been exported or not can be done by retrieving the transfer resource and looking for the [`cancel`](https://docs.dwolla.com/#cancel-a-transfer) link in the response -- if the transfer has exported, the `cancel` link is removed.
+If a funding-source has an **outgoing** transfer, we recommend not removing it until the transfer has been [exported](https://developers.dwolla.com/concepts/transfer-processing-times). Doing so will result in the cancellation of the transfer. Determining if a transfer has been exported or not can be done by retrieving the transfer resource and looking for the [`cancel`](https://developers.dwolla.com/api-reference/#cancel-a-transfer) link in the response -- if the transfer has exported, the `cancel` link is removed.
```

Comment:
This one needs to be linked to https://developers.dwolla.com/api-reference/transfers/cancel